docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -125,6 +134,73 @@ show up in ``DESCRIBE <table-name>`` or ``SELECT *``.
 
 This property is optional; the default is ``true``.
 
+``kafka.security-protocol``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Protocol used to communicate with brokers.
+Valid values are: PLAINTEXT, SSL.
+
+This property is optional; default is ``PLAINTEXT``.
+
+``kafka.ssl.keystore.location``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Location of the keystore file used for connection to Kafka cluster.
+
+This property is optional.
+
+``kafka.ssl.keystore.password``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Password for the keystore file used for connection to Kafka cluster.
+
+This property is optional, but required when ``kafka.ssl.keystore.location`` is given.
+
+``kafka.ssl.keystore.type``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+File format of the keystore file.
+Valid values are: JKS, PKCS12.
+
+This property is optional; default is ``JKS``.
+
+``kafka.ssl.truststore.location``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Location of the truststore file used for connection to Kafka cluster.
+
+This property is optional.
+
+``kafka.ssl.truststore.password``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Password for the truststore file used for connection to Kafka cluster.
+
+This property is optional, but required when ``kafka.ssl.truststore.location`` is given.
+
+``kafka.ssl.truststore.type``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+File format of the truststore file.
+Valid values are: JKS, PKCS12.
+
+This property is optional; default is ``JKS``.
+
+``kafka.ssl.key.password``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Password for the private key in the keystore file used for connection to Kafka cluster.
+
+This property is optional, but required when the Kafka broker has set ``ssl.key.password`` and two-way authentication is configured.
```

Comment:
Shouldn't it be `ssl.client.auth=required`?

Reply:
Yes you are right. I corrected the doc.